pyrene/src/components/TextArea/textArea.css

```diff
@@ -74,7 +73,7 @@
 
   font-family: FiraGO, Helvetica, sans-serif;
   font-size: 12px;
-  line-height: 12px;
+  line-height: 14px;
```

Comment:
It makes it to 32px with 1 row only (that's good), but it doesn't feel right, because when we add up multiple rows, the size of the whole is not a multiple of 8 anymore.
For me, we should have it to 16px, and padding 7 (+1 for the border).

Reply:
✅